### PR TITLE
fix: errors in lxqt-config-input_fr.ts file format

### PR DESCRIPTION
**Pull request by kyrios123**
add: initial version of lxqt-config-brightness_fr.ts
add: initial version of lxqt-config-monitor_fr.ts

### CODE REVIEW

**Comment by kyrios123** [comment]
lxqt-config : updates on french translations

File: lxqt-config/lxqt-config-input/lxqt-config-input_fr.ts [hunk 1]
```diff
@@ -53,18 +53,19 @@
...
         <source>Repeat interval:</source>
-        <translation type="unfinished">Intervalle de répétition :</translation>
+        <translation>Délai de répétition :</translation>
     </message>
     <message>
         <location filename="../../../keyboardconfig.ui" line="129"/>
         <source>Type in the following box to test your keyboard settings</source>
-        <translation type="unfinished">Tapez dans le champ ci-dessous pour tester vos paramètres clavier</translation>
+        <translatorcomment>Tapez dans le champ ci-dessous pour tester vos paramètres clavier</translatorcomment>
+        <translation type="unfinished">Saisissez une phrase pour tester les réglages</translation>
```
Comment (palinek) [inline lxqt-config/lxqt-config-input/lxqt-config-input_fr.ts:62]: Is the `type="unfinished"` used intentionaly?
(repeats multiple times)

Comment (kyrios123) [reply lxqt-config/lxqt-config-input/lxqt-config-input_fr.ts:62]: it comes by default from qt-linguist when i don't explicitly validate the translation.
when I am not 100% sure it's correct, I prefer leaving it as unfinished so someone else can still review it.

**Comment by palinek** [comment]
Did you close the PR intentionaly?

**Comment by kyrios123** [comment]
nope, sorry.. I reopened it

**Comment by palinek** [comment]
landed in @2737248, thanks

**Pull request closed without merging**